game/scripts/npc/items/farming/item_greater_tranquil_boots_2.txt

```diff
@@ -116,6 +117,18 @@
 				"var_type"				"FIELD_INTEGER"
 				"regen_from_creeps"		"20 30 45 60 120"    // New Functionality
 			}
+			// setting either of these to 0 will disable the functionality
+			// of creeps "bleeding" from attacking the wearer
+			"12"
+			{
+				"var_type"				"FIELD_FLOAT"
+				"creep_sap_damage"		"1.6 1.9 2.1 2.5 5"
+			}
+			"13"
+			{
+				"var_type"				"FIELD_FLOAT"
+				"creep_sap_duration"	"2.0"
```

Comment:
This is a float, but since the timer is on 1 second, any decimal has no effect. Should be integer for clarity, or could should respect decimal values.